overview/comparison-to-others.md

```diff
@@ -64,6 +66,26 @@ i18next.init({
 });​
 i18next.t('hello world'); // hallo Welt
 ```
+{% endtab %}
+
+{% tab title="TypeScript" %}
+```typescript
+import i18next from 'i18next';​
+
+i18next.init({
+    lng: 'de',
+    resources: {
+        de: {
+            translation: {
+                "hello world": "hallo Welt"
+            }
+        }
+    }
+});​
+i18next.t($ => $['hello world']); // hallo Welt
```

Comment:
why is this `$[`  ?

Reply:
maybe this should be mentioned in the docs?

Reply:
Hmm, I think we can rely on autocompletion to guide developers here:

![i18next-autocompletion-example](https://github.com/user-attachments/assets/59c470b3-07e0-401a-8b1d-ee33b985afe5)

Since this is how property access works in JavaScript in general, I don't expect users will be surprised by this behavior.

I'm happy to add it if you still think we should, though.